2019-11-19 21:01:06.365 - [Info] MITM Version: 1.26
2019-11-19 21:01:06.371 - [Info] Auto Access Enabled: true
2019-11-19 21:01:06.372 - [Debug] [Init] Auto Access Barrier: true
2019-11-19 21:01:06.392 - [Info] SSH man-in-the-middle server for 172.20.0.2 listening on 0.0.0.0:10101
2019-11-19 21:28:44.260 - [Debug] [Connection] Attacker connected: 36.75.142.161 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-19 21:28:45.901 - [Debug] [Auth] Attacker 36.75.142.161 trying to authenticate with "password"
2019-11-19 21:28:45.902 - [Debug] [Auto Access] Attacker: 36.75.142.161, Threshold: 1, Attempts: 1
2019-11-19 21:28:45.902 - [Debug] [Auto Access] Compromising the honeypot
2019-11-19 21:28:45.902 - [Debug] [Auto Access] Adding the following credentials: "root:u7y6t5r4e3w2"
2019-11-19 21:28:46.078 - [Debug] [Auth] Valid credentials - Password Authentication
2019-11-19 21:28:46.079 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.2
2019-11-19 21:28:46.128 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2019-11-19 21:28:47.202 - [Debug] [Connection] Attacker connected: 36.75.142.161 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-19 21:28:48.843 - [Debug] [Auth] Attacker 36.75.142.161 trying to authenticate with "password"
2019-11-19 21:28:48.844 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2019-11-19 21:28:48.845 - [Debug] [Auth] Attacker: 36.75.142.161 has so far made 1 attempts. Remaining: 5 attempts
2019-11-19 21:28:49.406 - [Debug] [Connection] Attacker closed the connection
2019-11-19 21:28:49.921 - [Debug] [EXEC] Noninteractive mode attacker command: cat /proc/cpuinfo | grep name | wc -l
2019-11-19 21:29:55.460 - [Debug] [Connection] Attacker closed connection
2019-11-19 21:29:55.476 - [Debug] [LXC] Container's OpenSSH server ended connection
2019-11-19 21:29:55.476 - [Debug] [LXC] Container's OpenSSH server closed connection
